### PR TITLE
[NETCore] Add docker medium post

### PR DESCRIPTION
**Pull request by gussiciliano**
Add docker medium post

### CODE REVIEW

File: infrastructure/docs/docker/README.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ## Docker guide
 
 First you need to install Docker (https://docs.docker.com/engine/installation/linux/docker-ce/ubuntu/#install-docker-ce-1)
-When you have installed Docker you need to create a Dockerfile, its content will depend on the technology, for example, in .NET Core we use [this definition](https://github.com/Wolox/netcore-bootstrap/blob/master/NetCoreBootstrap/Dockerfile) and in RoR we use [this definition](https://github.com/Wolox/rails-bootstrap/blob/master/Dockerfile)
+When you have installed Docker you need to create a Dockerfile, its content will depend on the technology, for example, in .NET Core we use [this definition](https://github.com/Wolox/netcore-bootstrap/blob/master/src/Dockerfile) and in RoR we use [this definition](https://github.com/Wolox/rails-bootstrap/blob/master/Dockerfile)
```
Comment (dtilve) [inline infrastructure/docs/docker/README.md]: I'd say
"Once you have Docker installed, you'll need to create a Dockerfile. Its content will depend on the technology, for example,.... "

Comment (gussiciliano) [reply infrastructure/docs/docker/README.md]: cool :D

**Comment by gussiciliano** [comment]
@mishuagopian don't forget thisPR :P

**Comment by gussiciliano** [comment]
@mishuagopian don't forget this PR xD

**Pull request merged**